addons/mail_bot/static/src/bugfix/bugfix.js

```diff
@@ -1,10 +1,8 @@
+/** @odoo-module **/
```

Comment:
bugfix files can all be removed (new manifest system allows to add new files in stable)

Reply:
Maybe out of scope of current PR, ok to do it separately.
I'm just mentioning it because I see it.

Reply:
task for it: https://www.odoo.com/web#id=2510653&action=333&active_id=1519&model=project.task&view_type=form&cids=1&menu_id=4720